Dockerfile

```diff
@@ -15,6 +15,12 @@ RUN apk add --no-cache libpq-dev tzdata shared-mime-info libc6-compat && \
     bundle config set with 'assets production' && \
     bundle config set deployment 'true' && \
     bundle install --no-cache --jobs=5 --retry=3 && \
+    mkdir -p app/javascript/assets/images/ && \
```

Comment:
We should not be doing this as the et_gds_design_system gem should sort out all assets for us - we shouldnt even really know or care that it uses govuk-frontend under the hood.  No other type of assets are done like this, so we just need to follow what other assets do in that gem